pkg/resource/deploy/import.go

```diff
@@ -169,7 +169,7 @@ func (i *importer) getOrCreateStackResource(ctx context.Context) (resource.URN,
 	typ, name := resource.RootStackType, fmt.Sprintf("%s-%s", projectName, stackName)
 	urn := resource.NewURN(stackName.Q(), projectName, "", typ, tokens.QName(name))
 	state := resource.NewState(typ, urn, false, false, "", resource.PropertyMap{}, nil, "", false, false, nil, nil, "",
-		nil, false, nil, nil, nil, "", false, "")
+		nil, false, nil, nil, nil, "", false, "", nil, nil)
```

Comment:
Should we not set created/modified to Now here?

Reply:
I have all of the code for setting timestamps in the `step.go`'s `*Step.Apply()` method `complete()` return callback for consistency.

Reply:
Sure but it looks like this code is adding things _directly_ to state so there isn't an associated step for these resources?

Reply:
Oh never mind I see where they're getting steps called on them now.